src/Features/Core/Portable/PopulateSwitch/AbstractPopulateSwitchCodeFixProvider.cs

```diff
@@ -146,7 +146,7 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
                 return;
 
             var model = await document.RequireSemanticModelAsync(cancellationToken).ConfigureAwait(false);
-            var switchStatement = (TSwitchOperation)model.GetOperation(switchNode, cancellationToken);
+            var switchStatement = (TSwitchOperation)model.GetOperation(switchNode, cancellationToken)!;
```

Comment:
📝 This is fine for now, but can you file an issue for us to follow-up with a `GetRequiredOperation` (or similar) method?

Reply:
[https://github.com/dotnet/roslyn/issues/40505](https://github.com/dotnet/roslyn/issues/40505)

---
In reply to: [360003645](https://github.com/dotnet/roslyn/pull/40493#discussion_r360003645) [](ancestors = 360003645)